mmdet/core/hook/wandblogger_hook.py

```diff
@@ -235,7 +235,6 @@ def after_train_epoch(self, runner):
             # Log the table
             self._log_eval_table(runner.epoch + 1)
 
-    @master_only
```

Comment:
By removing this decorator, the method will be called in every process however the wandb run is initialized only in the master process ([source](https://github.com/open-mmlab/mmcv/blob/6a03918f55c1cc860bcb4d64a0e868ca13cf08ba/mmcv/runner/hooks/logger/wandb.py#L78)). 

I think it should throw error. 

Plus to handle distributed training properly we would be using `wandb_service` ([link](https://docs.wandb.ai/guides/track/advanced/distributed-training#wandb-service)) once it's available as default with the [client](https://github.com/wandb/client/search?q=service).